### ISSUE TITLE
latest dev on android via termux

### ISSUE THREAD

**Issue by mvfsullivan** [AHAAAAAAA/PokemonGo-Map#3038]
Has anyone been able to successfully run the map on android?
Im running one of the pre-grunt builds, since I cant seem to get nodejs and grunt working in termux.
I can of course get the map to work via pc, and i could also host the map on a server, but it would be much easier to control directly from my phone. Please let me know!

**Comment by simoher** [comment]
U can use the android port :)

**Comment by mvfsullivan** [comment]
Is it possible to build the android port within android? I wont be near a computer for a few days at the least xD

**Comment by HibikiFox** [comment]
there is seriously a branch project that has an APK of this project... for android

like.... 

> Android Version
> 
> There is an Android port in the works. All Android related prs and issues please refer to this repo.

that is right on the main page

**Comment by mvfsullivan** [comment]
Where the hell are you seeing an APK?? I am not seeing anything.. Just the .zip

**Comment by HibikiFox** [comment]
.... wo there son I scrolled down the page I figured you should too. don't make it personal when I am trying to encourage you to find some simple answers for yourself.

**Comment by mvfsullivan** [comment]
Fair enough. I didnt think there would be a different dev branch that had an apk, since the main page said that it was a work in progress. Thanks for the help!
